### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by kryr34**

### CODE REVIEW

**Review by NOOBDY — APPROVED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (Error0229) [inline .github/ISSUE_TEMPLATE/bug_report.md:1]: Should we have a label set

File: .github/ISSUE_TEMPLATE/feature_request.md
Comment (Error0229) [inline .github/ISSUE_TEMPLATE/feature_request.md:1]: Do we need another template for requesting "I'm going to implement some feature"

Comment (kryr34) [reply .github/ISSUE_TEMPLATE/feature_request.md:1]: Please kindly submit your feature request and consider assign it to yourself.

**Review by Error0229 — APPROVED** [review]
LGTM

**Pull request merged**